projects/storefrontlib/src/lib/routing/store/reducers/router.reducer.ts

```diff
@@ -104,9 +106,17 @@ export const reducerProvider: Provider = {
   useFactory: getReducers
 };
 
-export const getRouterState: MemoizedSelector<any, any> = createFeatureSelector<
+export const getRouterFeatureState: MemoizedSelector<
+  any,
+  any
+> = createFeatureSelector<
   fromNgrxRouter.RouterReducerState<ActivatedRouterStateSnapshot>
->('router');
+>(ROUTING_FEATURE);
+
+export const getRouterState: MemoizedSelector<any, any> = createSelector(
+  getRouterFeatureState,
+  (state: any) => state.router
```

Comment:
I spent a while to understand why this new selector was introduced. And maybe it would be easier for me to understand from the beginning, if I had saw here:
```js
(state: any) => state[ROUTING_FEATURE]
```
But it can stay as is.